SingularityService/src/main/java/com/hubspot/singularity/mesos/SingularityMesosTaskBuilder.java

```diff
@@ -381,6 +381,10 @@ private void prepareCustomExecutor(final TaskInfo.Builder bldr, final Singularit
   private void prepareCommand(final TaskInfo.Builder bldr, final SingularityTaskId taskId, final SingularityTaskRequest task, final Protos.Offer offer, final Optional<long[]> ports) {
     CommandInfo.Builder commandBldr = CommandInfo.newBuilder();
 
+    if (task.getDeploy().getUser().isPresent()) {
+      commandBldr.setUser(task.getDeploy().getUser().get());
```

Comment:
We already had `customExecutorUser` for this purpose, but it had been deprecated for a while now. I've officially removed it in favor of the plain `user` field which will apply to the custom and default executor cases.